components/General/Faram/validator.js

```diff
@@ -1,10 +1,16 @@
-import { isFalsy, isTruthy } from '../../../utils/common';
+import {
+    isFalsy,
+    isTruthy,
+    findDifferenceInList,
+} from '../../../utils/common';
 
 const emptyObject = {};
 const emptyArray = [];
 
+const isList = item => Array.isArray(item);
+
 const isObject = item => (
```

Comment:
`isObject` is already defined in `utils/common`. And it is probably best to move `isList` to utils as well.